tests/utils.py

```diff
@@ -3,7 +3,7 @@
 
 from .forms import TestForm
 
-DJANGO3 = get_version() >= "3"
+DJANGO_VERSION = int(get_version()[:1])
```

Comment:
Why not use the package's `DJANGO_VERSION`?

Reply:
I'll do that, good catch.